### ISSUE TITLE
Implement Memory Map Feature

### ISSUE THREAD

**Issue by PropGit** [parallaxinc/Parallax-IDE#113]
[This is a placeholder so we don't forget to implement this feature.  The details will be placed here before we attempt solving this issue.]
- F8 or Ctrl-M - Tokenize and open Memory Map display.

**Comment by urbantumbleweed** [comment]
There is a good starting point with the legacy system to visualize the data.  It would be interesting to audit the existing and see if modern convention affords us opportunity to refresh the view.

**Comment by PropGit** [comment]
Yes, especially because we are also wanting this to support the Propeller in the future.  We will have to make a mock up and list functionality that we can discuss.
